docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+On-the-fly Pixel analysis
+-------------------------
+
+Click on the :icon:`fa fa-chart-area` button to start the plotting tool (1). Move the pointer to the main map, the pointer will be transformed into a :icon:`fa fa-plus` (2). Click anywhere in the AOI to plot data for this specific location in the following popup window.
+
+The plotting area (3) is dynamic and can be customized by the user.
+
+You can select the observation feature by selecting one of the available measures in the dropdown selector in the top left corner (4). The available bands are the same as the described previously.
+
+Using the slider (5), the temporal width displayed can be changed. It cannot exceed the start and/or end date of the time series.
+
+On the main graph, the orange lines shows the CCDC modelled time-series. Each of the blue points represents an actual observation. You can both hover over the point or the line to let the tooltip describe the value and date of the observation, as well as the model values and the temporal extent of the specific segment.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/ccdc_pixel_analysis.png
+    :title: Pixel Analysis
+    :width: 100%
+    :group: ccdc-asset-recipe
+
+.. warning::
+
+    The plot feature is retrieving information from GEE on the fly and serving it in an interactive window. This operation can take time depending on the number of available observations and the complexity of the selected pre-processing parameters. If the popup window displays a spinning wheel, wait up to 2 min to see the data displayed.
+
+
+Export
+------
+
+Trigger the export task
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Click on the :icon:`fa fa-cloud-download-alt` button to open the export dialogue. Here you can select the bands to retrieve and the scale at which you would like to save the asset. CCDC Assets are only compatible with Google Earth Engine, for which a new asset will be created in your personal Earth Engine repository.
+
+If the area covered is relatively small and you have enough storage quota left, you can generously select most of the bands relevant for land applications as shown in the below figure on the left. If you are more constrained by storage you will need ot decide on a subset of bands, for which the below figure on the right is a suggested starting point.
+
+The scale parameter depends on the data selected and the level of detail you will need for your further analysis. Landsat based assets are usually created at 30 meters. Sentinel-1 and 2 can be at 10 meter, but will need 9 times more space as compared to 30 meter resolution.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/ccdc_export_full.png
+    :title: Export CCDC Asset - full band selection
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/ccdc_export_reduced.png
+    :title: Export CCDC Asset - reduced band selection
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+
+Exportation status
```

Comment:
Do you think this section should go to features ?

Reply:
Exportation status, and how to check on earth engine ( I didn't added this so far)